src/languages/asciidoc.js

```diff
@@ -107,18 +107,38 @@ export default function(hljs) {
         begin: '^(NOTE|TIP|IMPORTANT|WARNING|CAUTION):\\s+',
         relevance: 10
       },
-      // inline strong
+      // inline unconstrained strong
+      {
+        className: 'strong',
+        begin: /\*{2}[\w\s\n]+/,
```

Comment:
So word space or newline... what is this mean to be excluding again?

Reply:
When you have something like this that isn't obvious at a glance usually adding a comment can help future maintainers.

Reply:
The code was overly complex and not even correct... let's forget about it, I was just tinkering :factory_worker: